timemachine/fe/single_topology.py

```diff
@@ -65,6 +65,9 @@ def setup_dummy_interactions_from_ff(ff, mol, dummy_group, root_anchor_atom, nbr
     bond_params, hb = top.parameterize_harmonic_bond(ff.hb_handle.params)
     angle_params, ha = top.parameterize_harmonic_angle(ff.ha_handle.params)
     improper_params, it = top.parameterize_improper_torsion(ff.it_handle.params)
+    chiral_atom_potential, chiral_bond_potential = top.setup_chiral_restraints(chiral_atom_k)
+    chiral_atom_idxs = chiral_atom_potential.potential.idxs
+    chiral_atom_params = chiral_bond_potential.params
```

Comment:
(recording from offline comments from @proteneer )

I introduced this typo in https://github.com/proteneer/timemachine/pull/1255/commits/e44f4e7788975d17e85494d3dd59c976b42c26b9 but didn't see it

```suggestion
    chiral_atom_params = chiral_atom_potential.params
```

Reply:
fixed in https://github.com/proteneer/timemachine/pull/1255/commits/efabaa21dc081104b0bddb4cb201cb18b0a0a47b